gulpfile.js

```diff
@@ -40,56 +85,114 @@ var gulp = require('gulp'),
         scripts: []
     };
 
-/* CSS - LESS */
+/* -------------------------------------------------------------------------------------------------
+ * processCss
+ * Accepts: a stream upon which to perform gulp stuff, and a message to display
+ * Returns: the processed stream
+ * ---------------------------------------------------------------------------------------------- */
 function processCss(inputStream, taskType) {
     return inputStream
-        .pipe($.plumber(function(error) {
+        .pipe($.plumber(function (error) {
             $.util.log($.util.colors.red('Error (' + error.plugin + '): ' + error.message + '\n'));
             this.emit('end');
         }))
-        .pipe($.newer(paths.styles.dest))
+        .pipe($.newer(paths.styles.core.dest))
         .pipe($.sass({ includePaths: paths.styles.src + 'imports/' }))
         .pipe($.sourcemaps.init())
             .pipe($.minifyCss({ advanced: false }))
             .pipe($.rename({ suffix: '.min' }))
         .pipe($.sourcemaps.write('./', { includeContent: true }))
-        .pipe(gulp.dest(paths.styles.dest))
+        .pipe(gulp.dest(paths.styles.core.dest))
         .pipe(browserSync.stream())
         .pipe($.if(flags.notify, $.notify({ message: taskType + ' task complete' })));
-}
+} // /function processCss
 
-/*
- * Accepts a gulp stream, gulp-uglify options, filename
- * - Concat stream content, minify and write to JS src/dest
- */
-function minifyJS (sourceStream, uglifyOptions, filename) {
+/* -------------------------------------------------------------------------------------------------
+ * minifyJS
+ * Accepts: a gulp stream, gulp-uglify options, libType, filename
+ * Returns: the processed stream
+ * Concat stream content, minify using options and write to JS src/dest using filename
+ * ---------------------------------------------------------------------------------------------- */
+function minifyJS (sourceStream, uglifyOptions, libType, filename) {
```

Comment:
minifyJS now accepts `libType` - must be one of either `core` or `vendor`, to match the paths set out in paths.scripts/styles[libType]...

e.g. `paths.scripts.core.src`, `paths.styles.vendor.dest` etc.